js/index.js

```diff
@@ -37,60 +37,67 @@ const speakersList = [
   },
 ];
 
-const hamburgerMenuBtn = document.querySelector(".hamburger-menu");
-const closeMobileMenuBtn = document.querySelector(".close-menu");
-const mobileMenu = document.querySelector(".mobile-menu-wrapper");
+const hamburgerMenuBtn = document.querySelector('.hamburger-menu');
+const closeMobileMenuBtn = document.querySelector('.close-menu');
+const mobileMenu = document.querySelector('.mobile-menu-wrapper');
```

Comment:
Great design @thasipablo 🤝

- [ ] [Optional] Kindly ensure there is an option to navigate to the home page from the About page of the mobile view.
![mobile-about](https://github.com/thasipablo/around_the_table/assets/63917725/a7dd17a9-d723-40ae-8fd1-1815ae26979f)

Reply:
Thank you for your time reviewing my code. I am going to add it.
Thank you.